ts/features/itwallet/common/components/ItwEidInfoBottomSheetContent.tsx

```diff
@@ -52,6 +56,14 @@ const ItwEidInfoBottomSheetContent = ({ navigation }: Props) => {
       });
     };
 
+    useEffect(() => {
+      // Passing valid hardcoded since the credential should always be valid when visiting this bottomsheet
```

Comment:
After PR #6264 the eID bottom sheet could be opened with an expired eID. Let's keep this in mind (I am not requesting changes here).

Reply:
Based on that PR i think it would be nice to use the new selector added in that PR `itwCredentialsEidStatusSelector` instead of using the hardcoded "valid" value to track the correct status inside mixpanel. What do you think?

Reply:
I agree. Let's see which one will be merged first, then we'll update this component.

Reply:
Since https://github.com/pagopa/io-app/pull/6264 has been merged, i updated this PR with the id status mapping.